koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -376,6 +378,24 @@ def _incremental_upload_logs(self, child_pid=None):
         finally:
             watcher.clean()
 
+    def _upload_logs_once(self):
```

Comment:
I'm trying hard, but the only difference I see here comparing to `_incremental_upload_logs(child_pid=None)` is `except koji.ActionNotAllowed`

Is that exception the only reason to not reuse existing function?